pymc/step_methods/arraystep.py

```diff
@@ -181,7 +181,13 @@ def metrop_select(mr: np.ndarray, q: np.ndarray, q0: np.ndarray) -> tuple[np.nda
 
     Parameters
     ----------
-    mr: float, Metropolis acceptance rate
+    mr: float, Metropolis acceptance rate. In theory, this acceptance rate is the
```

Comment:
Can this be summarized a bit more. Isn't it enough to say it's the logarithm of the ratio of probabilities?

Reply:
In places where it is called, the variable passed in here is typically called `delta_logp`. I wanted to explain the parameter in a way that explains the origin of the name.

Reply:
I understand but it's too verbose in my opinion. Logarithm of probability ratio is pretty explicit. If you look in the database you'll quickly get used to seeing everything working on log scales

Reply:
The parameter description should be (and should have been) on a different line, with only the parameter name and its type in the first line:

```
mr : float
    description
    which
    can be
    multiline
```

it looks like this class isn't on the website listed api docs so not fixing won't result in rendering errors unless/until it is added to the website.